content/docs-v1/citation-bibliographies.md

```diff
@@ -33,72 +33,110 @@ entries:
+
+If you want to suppress the page level bibliography but keep the in-text citations, go into the `content/_data/config.yaml` and change the value to `false`:
+
+```yaml
+  bibliography:
+    displayOnPage: false
+```
 
-Your publication will automatically include a page-level bibliography listing all works that were cited on that page using the `q-cite` shortcode. A heading can be customized to go above the page bibliography with the `biblioHeading` parameter in your `config.yaml` file. The default is "Bibliography".
+If you do not want to use in-text citations but still want to include a page-level bibiliography you have a two options:
 
-Additionally, to create a complete bibliography for your entire publication, from all the entries in the project's `references.yaml` file, you can create a `bibliography.md` file and use the `q-bibliography` shortcode. The resulting bibliography will be output as a page in your publication. By default, your bibliography will automatically be ordered according to the `full` value in your `references.yaml` file.
+- Add a page-level biblio using Markdown
+- List the relevant IDs from the `references.yaml` file in the page YAML. Keep in mind, this may be unwieldy if there are a lot of references. You would then use the `{% bibliography %}` shortcode to add the page-level bibliography at the end of the essay.
 
-```go
-{{</* q-bibliography */>}}
+```yaml
+title: Chapter Title
+weight: 10
+layout: page
+references:
+  - "West 1939"
+  - "Evans 1938"
+  - "Lynd 1929"
 ```
 
-{{< q-figure id="modern-bibliography" >}}
+For the latter option, the order they’re listed in doesn’t matter. Quire will sort them all alphabetically unless a `sort_as` attribute has been added in the `references.yaml` file.
 
+### Complete Bibliography
 
-### Display the Short Reference in Bibliographies
+Additionally, to create a complete bibliography for your entire publication, from all the entries in the project's `references.yaml` file, you would add a `bibliography.md` file to your `content` directory. Then insert the `{% bibliography %}` shortcode to this `.md` file. The resulting bibliography will be output as a page in your publication. By default, your bibliography will automatically be ordered according to the `full` value in your `references.yaml` file unless the `sort_as` attribute has been applied.
+
+{{< q-figure id="modern-bibliography" >}}
 
-Bibliographies displayed automatically at the bottom of pages, and those generated with the `q-bibliography` shortcode, can be a list of the full version of the reference, or can include the short version as well. This is controlled globally (all bibliographies in the project have to be the same format) in the `config.yaml` file with the `displayBiblioShort` property, can be set to `"true"` or `"false"`.
+A heading can be customized to go above the page bibliography in your `content/_data/config.yaml` file. The default heading is "Bibliography".
```

Comment:
This part is still important to keep but applies to page-level bibliographies specifically